spec/db/db_file_spec.rb

```diff
@@ -1,14 +1,13 @@
-require 'db/db'
+require 'db/db_file'
 
-describe DB do
+describe DBFile do
 
-  let(:reader)  {instance_double(Reader).as_null_object}
-  let(:writer)  {instance_double(Writer).as_null_object}
-  let(:db)      {described_class.new(reader, writer)}
+  let(:handler) {instance_double(FileHandler).as_null_object}
```

Comment:
`handler` is not a particular good name as it doesn't really provide information what kind of object I'm working with.